skeleton/github/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-https://github.com/nimblehq/git-template/issues/??
+Close #
```

Comment:
```suggestion
- Close #
```

|Without the dash| With the dash |
|---|---|
|<img width="159" alt="image" src="https://user-images.githubusercontent.com/77609814/183597096-5cdf1168-81de-4f2c-8e22-e42a9bc8e738.png">|<img width="418" alt="image" src="https://user-images.githubusercontent.com/77609814/183597158-2861332f-25cd-4021-8fcb-0beb6bf9a075.png">|

Reply:
💯 That's good, changed in https://github.com/nimblehq/infrastructure-templates/pull/97/commits/c3fe87984b2aadd8b57c25720be25114d27ee4eb